src/main/java/net/minestom/server/entity/acquirable/AcquirableEntity.java

```diff
@@ -143,7 +143,7 @@ public void refreshChunkEntry(@NotNull ThreadProvider.ChunkEntry chunkEntry) {
             this.chunkEntry = chunkEntry;
         }
 
-        public BatchThread getBatchThread() {
+        public TickThread getBatchThread() {
```

Comment:
Should this be renamed to `getTickThread`?

Reply:
Effectively